app/src/main/java/org/dhis2/data/forms/dataentry/fields/section/SectionHolder.kt

```diff
@@ -138,4 +145,21 @@ class SectionHolder(
     fun setBottomShadow(showShadow: Boolean) {
         formBinding.shadowBottom.visibility = if (showShadow) View.VISIBLE else View.GONE
     }
+
+    fun setLastSectionHeight(previousSectionIsOpened: Boolean) {
+        val params = formBinding.lastSectionDetails.layoutParams
+        val finalHeight = if (previousSectionIsOpened) {
+            48.px
```

Comment:
can we use dp instead of px?

Reply:
It is using dp. The extension function calculates the value

Reply:
shouldn't it be 48.dp instead?

Reply:
I agree with @JaimeToca this tells me its 48 px. 😕

Reply:
I think the name is a bit confusing, talking to @Balcan  and it meant fromDpToPixel

Reply:
changed